04-cs-fundamentals/classroom/Sorting.md

```diff
@@ -1,19 +1,33 @@
 # Sorting
+
 ## Learning Goals
-- Understand common sorting algorithms and their efficiency
+
+By the end of this lesson you should be able to:
+
+- Describe common sorting algorithms and their efficiency
   - Bubble sort
   - Selection sort
   - Insertion sort
   - Merge sort
 
-### Notes
-In our last conversation on [Arrays](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/Arrays.md), we saw how having the data set sorted improved the algorithms we could use. e.g. <strong>Finding an element with minimum or maximum value</strong> went from a *O(n)* algorithm for unsorted array to an *O(1)* algorithm for a sorted array. Similarly, <strong>binary search</strong> which is possible on sorted data set is an *O(log n)* algorithm as compared to *O(n)* <strong>linear search</strong> algorithm. There are also other everday examples where we expect the data set to be sorted e.g. last names sorted from 'a' through 'z', words in a dictionary, test scores sorted in descending order, employee ids sorted in ascending order etc.
+## Video Lesson & Slides
+
+- [Video Lesson](https://adaacademy.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=a4668a63-9fb8-4778-b0a4-aaca006b34c8)
+- [Slide Deck](https://docs.google.com/presentation/d/1GkYP84Cbg3I5KS_wIfRN8Gn-5tQ_46vV5zWt1dTZn14/edit?usp=sharing)
+- [Sorting Exercise](https://github.com/Ada-C12/reverse-sentence) - link incoming
+
+## Introduction
+
+In our last lesson on [Arrays](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/Arrays.md), we saw how having the data set sorted improved the algorithms we could use. e.g. **Finding an element with minimum or maximum value** went from a *O(n)* algorithm for unsorted array to an *O(1)* algorithm for a sorted array. Similarly, **binary search** which is possible on sorted data set is an *O(log n)* algorithm as compared to *O(n)* **linear search** algorithm. There are also other everday examples where we expect the data set to be sorted e.g. last names sorted from 'a' through 'z', words in a dictionary, test scores sorted in descending order, employee ids sorted in ascending order etc.
```

Comment:
Not a big deal but we're generally pretty consistent about `_`s for italic/emphasis and `**` for bold/strong emphasis, I think because of Slack and it really helps me personally parse Markdown.

```suggestion
In our last lesson on [Arrays](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/Arrays.md), we saw how having the data set sorted improved the algorithms we could use. e.g. **Finding an element with minimum or maximum value** went from a _O(n)_ algorithm for unsorted array to an _O(1)_ algorithm for a sorted array. Similarly, **binary search** which is possible on sorted data set is an _O(log n)_ algorithm as compared to _O(n)_ **linear search** algorithm. There are also other everday examples where we expect the data set to be sorted e.g. last names sorted from 'a' through 'z', words in a dictionary, test scores sorted in descending order, employee ids sorted in ascending order etc.
```